suricata/package.yaml

```diff
@@ -1,33 +1,1226 @@
+---
 id: suricata
 name: Suricata
 author: Tenzir
 author_icon: https://github.com/tenzir.png
-package_icon: https://suricata.io/wp-content/uploads/2023/09/Logo-Suricata-vert-R.png
-
+package_icon: |
```

Comment:
Note that this adds a `\n` to the end of `package_icon`, so unless the platform is trimming whitespace this will cause the package icon to fail to load. Can you try this out by installing the package locally?

Reply:
I installed the package on my local node `swift` and the icon appears:

![image](https://github.com/user-attachments/assets/d6f4c7b7-2222-4e99-8080-5cd94a670104)